src/main/java/io/localmotion/picture/entity/PictureType.java

```diff
@@ -0,0 +1,5 @@
+package io.localmotion.picture.entity;
+
+public enum PictureType {
```

Comment:
Not sure if we need this, but it might be useful for usage stats. So keep it in, but change the types to USER_PROFILE, INITIATIVE and CHATBOX